### ISSUE TITLE
Create OASIS+ buyers' ordering guide (printable) page

### ISSUE THREAD

**Issue by rosamundtgov** [GSA/oasis-plus#449]

**Comment by rosamundtgov** [comment]
the pages that will be included in the buyers' ordering guide (printable) include:

- About OASIS+ 
-- Roles and responsibilities
-- Contract families 
--  Domains and labor categories 
--- NAICS code by domain
-- Roles and responsibilities 
- Research the OASIS+ contract features
-- Labor considerations  
-- Task orders 
-- Small business considerations 
-- Pricing and wage rates 
-- Prohibitions 
- Complete market research 
- Obtain a Delegation of Procurement Authority
- Develop and issue the solicitation
- Manage the task order
- Award the task order

**Comment by rosamundtgov** [comment]
will get the Research agencies and sellers (TBD) question answered Monday and update this ticket -

**Comment by weiwang-gsa** [comment]
@klohman when print, what is the print version looks like, does it have the cover page like it currently have? or no covering page at all? @rosamundtgov  just double check with you, the print version doesn't include Buyers guide landing page and request scope view, also the display order is changed, Manage the task order comes first and Award the task order the end?  Will the order same as menu (side nav ) order, or order is actually different?

**Comment by klohman** [comment]
@weiwang-gsa We can leave the cover sheet for the printable version. 

Also the order should match the menu - so Award the task order then Manage the task order. 

The [Scope Review page](https://github.com/orgs/GSA/projects/111/views/1?filterQuery=scope&pane=issue&itemId=54302611) seems to be on hold so I don't believe it will be included in the printable guide. We should check on the status of this page content at our weekly sync tomorrow though.

**Comment by rosamundtgov** [comment]
the order should match the menu. I updated the description.

**Comment by weiwang-gsa** [comment]
@klohman is the table of content still needed on this page?

**Comment by klohman** [comment]
@weiwang-gsa Yes this is an exception where we do want to still include the TOC.

**Comment by weiwang-gsa** [comment]
@klohman this is ready for design review here: https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/wwang/ordering-guide-new/oasis-plus/buyers-guide/

There are AMP issues as we combine all in one page, I've addressed heading level issues and other issues that are easy fixes. Below are the remaining AMP issue that need discussion.  

1. Same text with different link.

- TOC section is pointing to h2 headers on this page, and same text in other pages links to individual page, AMP doesn't like same text with different link. For this, should we ignore it or try to address it?

 ![Image](https://github.com/GSA/oasis-plus/assets/119954777/58fbd663-1f69-444a-aefb-20e3746f7d11)

 ![Image](https://github.com/GSA/oasis-plus/assets/119954777/15883cd1-4067-477d-8b7f-a268da9d946c)

-  Same text in different pages are not having same link, maybe we should make it consistent? 
  e.g.  Buy.gsa.gov:
 ![Image](https://github.com/GSA/oasis-plus/assets/119954777/408a73ac-dd35-407d-bd1d-3cbd51f02f31)
 ![Image](https://github.com/GSA/oasis-plus/assets/119954777/eb48c7b9-5bf1-4178-99b2-d96292e43564)

User Guid:
![Image](https://github.com/GSA/oasis-plus/assets/119954777/f8528907-6066-4465-9014-812edd00e32e)

I didn't list all of them, maybe easier just run AMP tool and see what it refers to. 

2. Domain Code table. strange it doesn't complain on the individual domain code page, but got complaints in the printable version. Will take a further look.

![Image](https://github.com/GSA/oasis-plus/assets/119954777/585c0004-f4ea-4caa-a7ea-788fcd1c6f0a)

Not sure if those AMP issue should be tracked in separate user story or not.

**Comment by weiwang-gsa** [comment]
@rosamundtgov for "Change to How to complete government registration (all instances should be sentence case)", I did that and complaints still remain, then I noticed it is not only the capitalization issue, the links are different, one linked to a video, one is not, so I actually include (video) in the link text. 

I put in [PR](https://github.com/GSA/oasis-plus/pull/492) for you to review, it is easier than me to explain what is been changed, let me know if you are fine with it, or you can leave comment in PR what you prefer to be changed.

**Comment by rosamundtgov** [comment]
Looks good to me! @PowderD @ekidenda ready for your review.

**Comment by ekidenda** [comment]
@rosamundtgov reviewed the updates and it looks good! Moving to done.
